docs/extend/modular-resources/examples/rplidar.md

```diff
@@ -47,10 +47,14 @@ brew tap viamrobotics/brews && brew install rplidar-module
 
 ## Configuration
 
-Physically connect the RPlidar to your machine.
-Go to your robot's page on the [Viam app](https://app.viam.com/).
+Physically connect the RPlidar to your robot.
 
-{{< tabs name="Add the Rplidar component">}}
+Be sure to position the RPlidar so that it faces forward in the direction your robot travels.
+For example, if you are using the RPlidar A1 model, mount it to your robot so that the pointed end of the RPlidar mount housing points in the direction of the front of the robot.
```

Comment:
do we need to link to where to find the A1 model? I think people may now know what we mean by that